all.json

```
@@ -1,6 +1,31 @@
 {
 	"allow": [],
 	"deny": [
+		"verify-mycollabland.com",
+		"connectcollabland.com",
+		"collablandy.cc",
+		"collablandt.cc",
+		"collablandq.cc",
+		"collablandoo.cc",
+		"collablandk.cc",
+		"collablandgt.cc",
+		"collablandd.com",
+		"collablandsecuredconnect.com",
+		"metamask-walletsynfix-247.square.site",
+		"dappsreapprove.xyz",
+		"coinsolutionnetwork.live",
+		"apikucoinconnect.com",
+		"filsfixmimas.com",
+		"node-debug.com",
+		"dappsvalidator.netlify.app",
+		"atomic.cryptlinkaffix.online",
```

```suggestion
		"atomic.cryptlinkaffix.online",
		"cryptlinkaffix.online",
```

Root level is also problematic.